src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/SimpleChimera.java

```diff
@@ -279,8 +279,13 @@ private boolean hasIncompletePicture() {
         return AssemblyContigWithFineTunedAlignments.hasIncompletePictureFromTwoAlignments(regionWithLowerCoordOnContig, regionWithHigherCoordOnContig);
     }
 
+    // TODO: 5/5/18 Note that the use of the following predicate is currently obsoleted by
```

Comment:
I'm a bit confused by this comment: this method is still being called in several places, so how is it obsolete?